docs/Deploy_FATE_Cluster_with_Admin_Role_in_Certain_Namespace.md

```diff
@@ -0,0 +1,138 @@
+
+When deploying KubeFATE and FATE cluster with Kubernetes, users don't always have the cluster-admin privilege of the whole K8s cluster. To deploy KubeFATE and run FATE jobs, [admin]([admin](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)) role provided by Kubernetes is necessary in certain namespace. And in this case, some configuration files need to be modified.
+
+## Deploy Steps
+
+### Environment
+
+Start with a K8s cluster which does not have KubeFATE preinstalled, but have 3 namespaces (`fate-exchange`，`fate-9999`，`fate-10000`) and 2 users:
+
+- User `9999` which bound with `admin` role in namespace `fate-9999` and `fate-exchange`
+- User `10000` which bound with `admin` role in namespace `fate-10000`
+
+Similar to [Deploy an exchange central multi parties federated learning network with KubeFATE](https://github.com/FederatedAI/KubeFATE/wiki/Deploy-an-exchange-central-multi-parties-federated-learning-network-with-KubeFATE), in this document we will deploy two Parties with an exchange and run FATE test job.
+
+| party       | party ID | owner      | namespace     | K8s version | KubeFATE version | FATE version |
+|-------------|----------|------------|---------------|-------------|------------------|--------------|
+| exchange    | 1        | user-9999  | fate-exchange | v1.24.3     | v1.4.4           | v1.8.0       |
+| party-9999  | 9999     | user-9999  | fate-9999     | v1.24.3     | v1.4.4           | v1.8.0       |
+| party-10000 | 10000    | user-10000 | fate-10000    | v1.24.3     | v1.4.4           | v1.8.0       |
+
+### Install KubeFATE
+
+Because there is no KubeFATE in the cluster, each user needs to [install KubeFATE](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#readme) separately. Default configuration files needs to be modified beforehand.
+
+User `9999` will use the `rbac-config.yaml` file below:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubefate-admin
+  namespace: fate-9999
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sc-9999-edit-binding
+  namespace: fate-exchange
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
+subjects:
+  - kind: ServiceAccount
+    name: kubefate-admin
+    namespace: fate-9999
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sc-edit-binding
+  namespace: fate-9999
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
+subjects:
+  - kind: ServiceAccount
+    name: kubefate-admin
+    namespace: fate-9999
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubefate-secret
+  namespace: fate-9999
+type: Opaque
+stringData:
+  kubefateUsername: admin
+  kubefatePassword: admin
+  mariadbUsername: kubefate
+  mariadbPassword: kubefate
+```
+
+User `10000` use the `rbac-config.yaml` similar to user `9999` but doesn't create rolebinding in `fate-exchange` namespace.
+
```

Comment:
Could you further add some explanation of the modification to the rbac config? like "In general, we need to remove any cluster scope role and rolebindings and create only role and role bindings in the corresponding namespaces"